share-coordinator/src/main/java/org/apache/kafka/coordinator/share/ShareCoordinatorService.java

```diff
@@ -539,7 +539,7 @@ public void onNewMetadataImage(MetadataImage newImage, MetadataDelta delta) {
     }
 
     private TopicPartition topicPartitionFor(SharePartitionKey key) {
```

Comment:
Can we add a unit test for this method?

Reply:
Is the string representation `String.format("%s:%s:%d", groupId, topicId, partition)` included by KIP-932? if not, maybe we should add it to the KIP and json file (`FindCoordinatorRequest`).

Reply:
I have added it to the JSON file. I'll make sure the KIP contains this too.

Reply:
Kip ref:
https://cwiki.apache.org/confluence/plugins/servlet/mobile?contentId=255070434#KIP932:QueuesforKafka-Thesharecoordinatorandtheshare-groupstatetopic